Lombiq.Tests.UI/Extensions/TypedRouteUITestContextExtensions.cs

```diff
@@ -43,11 +43,18 @@ public static class TypedRouteUITestContextExtensions
         this UITestContext context,
         Expression<Action<TController>> actionExpression,
         params (string Key, object Value)[] additionalArguments)
-        where TController : ControllerBase =>
-        TypedRoute
-            .CreateFromExpression(actionExpression, additionalArguments, CreateServiceProvider(context))
+        where TController : ControllerBase
+    {
+#pragma warning disable CA2000 // Dispose objects before losing scope
```

Comment:
If we use dispose it will break the functionality as the caller method of this method will have a disposed object instead of a living one.